index.bs

```diff
@@ -407,4 +434,30 @@ This callback will be invoked when there are changes to <i>target</i>'s intersec
                 </ol>
             </ul>
         </ul>
+
+<h3 id='3-3-external-spec-integrations'>External Spec Integrations</h3>
+
+<h4 id='3-3-1-event-loop'>HTML Processing Model: Event Loop</h4>
+    An <a>Intersection Observer</a> processing step should take place AFTER the
+    layout and rendering steps have been performed in the
+    <a href="https://html.spec.whatwg.org/multipage/webappapis.html#processing-model-8">
```

Comment:
I actually imagine that the whole 3.3 section goes away, and these steps would literally get patched into the Webapps API spec. Because of that, I wanted to skip pulling the references up into the main <pre class=anchors> block so I don't forget to clean it up. Is that reasonable?